testng-eclipse-plugin/src/main/org/testng/eclipse/ui/SuiteRunInfo.java

```diff
@@ -113,7 +113,7 @@ public void onStart(TestMessage tm) {
   }
 
   public void onFinish(TestMessage tm) {
-    m_testCount++;
+//    m_testCount++;
```

Comment:
m_testCount is not changed, which comes from `genericMessage.getTestCount()` when `SuiteRunInfo.onInitialization(GenericMessage)`